app/components/autocomplete/at_mention/at_mention.tsx

```diff
@@ -230,6 +230,21 @@ const getAllUsers = async (serverUrl: string) => {
     return queryAllUsers(database).fetch();
 };
 
+const getUsersFromDMSorted = async (serverUrl: string, memberIds: string[]) => {
+    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+    if (!database) {
+        return [];
+    }
+
+    return database.get<UserModel>(USER).query(
+        Q.unsafeSqlQuery(`SELECT DISTINCT u.* FROM User u
+        INNER JOIN ${CHANNEL_MEMBERSHIP} cm ON cm.user_id=u.id
+        LEFT JOIN ${CHANNEL} c ON c.id=cm.id AND c.type='${General.DM_CHANNEL}'
+        LEFT JOIN ${MY_CHANNEL} my
```

Comment:
Why I left join? I expect that a DM channel has a `Channel` and `MyChannel` relation met. So this could be an inner join that should improve the query plan.